### PR TITLE
Feature/login/initial

### PR DESCRIPTION
**Pull request by rkline2**
This code consists of the user login/sign up page. Both implementations are located in a single html file. There's also another html file that consists of the password recovery page. 

This is front-end code only and still requires back-end maintenance to achieve full functionality.

### CODE REVIEW

File: Octogon_Login/pswrd_recv.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <meta property="og:title" content="">
+  <meta property="og:type" content="">
+  <meta property="og:url" content="">
+  <meta property="og:image" content="">
+
+  <link rel="manifest" href="site.webmanifest">
+  <link rel="apple-touch-icon" href="icon.png">
```
Comment (Squirrelofudun) [inline Octogon_Login/pswrd_recv.html]: icon.png doesn't currently exist.

File: Octogon_Login/pswrd_recv.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+          <h3>Please enter your personal credentials and continue your journey with us!</h3>
+        </div>
+			</div>
+		</div>
+  </div>
+</div>
+
+  <script type="text/javascript" src="js/scripts.js"></script>
+  <script src="js/plugins.js"></script>
+  <script src="js/main.js"></script>
```
Comment (Squirrelofudun) [inline Octogon_Login/pswrd_recv.html]: main.js throws an error

**Review by Squirrelofudun — CHANGES_REQUESTED** [review]
icon.png needs to be added, and main.js needs to be fixed/removed

File: Octogon_Login/pswrd_recv.html (hunk 1 above)
Comment (rkline2) [reply Octogon_Login/pswrd_recv.html]: Okay sounds good. Issue is fixed

File: Octogon_Login/pswrd_recv.html (hunk 2 above)
Comment (rkline2) [reply Octogon_Login/pswrd_recv.html]: Okay sounds good. Issue is fixed

**Pull request closed without merging**